### ISSUE TITLE
error configuring nextcloud sync

### ISSUE THREAD

**Issue by talon35** [quillpad/quillpad#115]
Hi.
I want to use the nextcloud sync feature (I'm aware that's experimental as for now).
However, I keep receiving error message "an error occurred" after entering my credentials. I tested the url in the browser and I'm sure that at least the credentials are valid.
Do I have to install something on my nextcloud server first? Couldn't find a Quillpad package, though.
Thanks for the help

**Comment by jsixface** [comment]
Yes, you need the [Notes app](https://apps.nextcloud.com/apps/notes) to be installed in your Nextcloud instance. The URL will be the root of your Nextcloud URL. I guess in your case it would be https://domain.de/nextcloud

**Comment by talon35** [comment]
Works like a charm! Thanks a lot.
